.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -13,16 +13,20 @@ body:
       description: |
         Please check the following before filing a bug report.
       options:
-        - label: This bug is in DelphiLint, not SonarDelphi, SonarQube, or my Delphi code.
+        - label: This bug is in DelphiLint, not SonarQube or the Delphi IDE.
           required: true
         - label: This bug has not already been reported.
           required: true
+        - label: >
+            This bug is not a problem related to analysis quality (e.g. parsing errors, false positive issues), which
+            should be raised on the [SonarDelphi repository](https://github.com/integrated-application-development/sonar-delphi/issues).
+          required: true
   - type: input
     attributes:
       label: Delphi IDE version
       description: >
         Specify your Delphi IDE version.
-      placeholder: "1.0.0"
+      placeholder: "Delphi X Version Y"
     validations:

```

Comment:
Let's give a real Delphi version as a placeholder here and encourage them to include the "edition" to catch wayward community edition users.

Reply:
Looks like we're encouraging the usage of the codenames here as well, which I don't think is necessary when the version number is quite clear by by itself.

Reply:
Updated now to use the latest Delphi 12 version as a placeholder. I think it might look a little [out of place](https://github.com/fourls/delphilint-test/issues/new?assignees=&labels=bug%2Ctriage&projects=&template=bug_report.yml), any thoughts?

Reply:
I think it's okay and makes it very clear what the expectation is in terms of detail and format.